src/core-manager/index.js

```diff
@@ -145,9 +151,10 @@ export class CoreManager extends TypedEmitter {
     }
 
     // Load persisted cores
-    const stmt = sqlite.prepare(`SELECT publicKey, namespace FROM ${TABLE}`)
-    // @ts-ignore - don't know types returned from sqlite here
-    for (const { publicKey, namespace } of stmt.all()) {
+    const rows = db.select().from(coresTable).all()
+    for (const { publicKey, namespace } of rows) {
+      // TODO: Does it make sense to just ignore here or should we throw an error?
+      if (!isKnownNamespace(namespace)) continue
```

Comment:
hm tried to do that but seems like it causes some issues with resolution e.g. when running tests:

```sh
ReferenceError: Cannot access 'NAMESPACES' before initialization
    at file:///Users/andrewchou/GitHub/digidem/mapeo-core-next/src/schema/project.js:36:40
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25)
ERROR: "test:unit" exited with 1.
```

Reply:
Ah right, yes this is because of the import loop between src/core-manager/index.js and src/schema/project.js (they each import each other). Moving the definition of `NAMESPACES` into a separate file (e.g. `src/constants.js` solves the problem. I don't think there's anything particularly sub-optimal of moving it there.

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/429/commits/83df1856d730d452800fe08c8f580a163d72ac4a